package.json

```diff
@@ -25,24 +25,24 @@
   "devDependencies": {
```

Comment:
Do you also need an increment to the package version for mindfulness so folks can upgrade?